monkey/monkey_island/cc/ui/src/components/attack/techniques/MitigationsComponent.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import ReactTable from 'react-table';
+import marked from 'marked';
+import '../../../styles/report/AttackReport.scss';
+
+
+class MitigationsComponent extends React.Component {
+
+  constructor(props) {
+    super(props);
+    if (typeof this.props.mitigations !== 'undefined'){
+      let descriptions = MitigationsComponent.parseDescription(this.props.mitigations.description);
+      this.state = {name: this.props.mitigations.name, descriptions: descriptions};
```

Comment:
![image](https://user-images.githubusercontent.com/48879847/77849562-4c484980-71d5-11ea-9e97-f1eda11dd354.png)
Mitigation -> Mitigations in the title of the table